### DISCUSSION TITLE
Redis Cluster Support

### DISCUSSION THREAD

**Discussion by rad-pat** [aio-libs-abandoned/aioredis-py#1241] [General]
I see that redis-py has recently included cluster support. As v2.0 is based upon redis-py, is there any mechanism in place to include modifications made to that library since the port over to asyncio? Is it reasonable to compare the two libraries for changes?

*** UPDATE ***
I totally missed PR https://github.com/aio-libs/aioredis-py/pull/1156 which brings in changes up to October.  Is this something that will be happening with regularity?

**Comment by Andrew-Chen-Wang** [comment]
Yes, it should happen more frequently than it currently is. Unfortunately, we've got some core issues that I'd like to get fixed first before merging new features in (though I think we should just merge in #1156). The good news is those features have been tested on the PR with updated tests, so if you'd like, just copy over the code for now and subclass `Redis`.

@seandstewart I think it's best we merge an update from redis-py like #1156 rather than wait for so long.

**Comment by Pritamhars** [comment]
@Andrew-Chen-Wang @seandstewart is Redis Cluster Support available yet in new release?

**Reply by Andrew-Chen-Wang** [reply]
no please follow #1267

if you'd like to help make it happen,'please contribute to the port
